src/extension.ts

```diff
@@ -48,6 +49,7 @@ export async function activate(context: vscode.ExtensionContext) {
     juliaexepath.activate(context)
     await juliaexepath.getJuliaExePath() // We run this function now and await to make sure we don't run in twice simultaneously later
     repl.activate(context)
+    setContext('Julia:activated', true)

```

Comment:
Not sure whether this is the right place to put the call; on my machine it works fine.